feature/metis/manage-conversations/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/ui/conversation/overview/ConversationOverviewViewModel.kt

```diff
@@ -432,4 +432,16 @@ class ConversationOverviewViewModel(
         isExpanded: Boolean,
         showPrefix: Boolean = true
     ) = ConversationCollection(this, isExpanded, showPrefix)
+
+    fun removeSectionPrefix(value: String): String {
```

Comment:
This method is only related to how the view displays its content. Therefore, I think it does not belong to the view model. Can you move it back to the view?